script/Ciel/Day 8/08_01_CIEL08_1C.txt

```diff
@@ -327,72 +353,79 @@ I understand how she feels.
 [sha:d0a5528802dddce2ba69f579521ee4f65134298f]{
 -- Page 94, Offset 18486.
 -- しかし、人体というものは時としてよく分からない仕組みで動くものなのです。
-However, humans can be swayed in mysterious ways.
+But, the human body just behaves in unpredictable ways sometimes.
 }
 [sha:be8f1594c88f372ac1217d7258640738c42c9062]{
 -- Page 95, Offset 18487.
 -- 「……その、繋がらない感じがイヤなんです。
-"...Umm, I don't like when things don't connect.
+"Um... I don't like it when things don't connect.
 }
 [sha:94d0a15621ff0334e76d830559d74dbe8bd5aee1]{
 -- Page 95, Offset 18488.
 -- 誰だってあるでしょ、理由はないけどダメなコトって。食べ物の好き嫌いと同じです」
-Other people are the same, I mean, having things they dislike for some unknown reason. It's the same as liking or disliking some food."
+Everyone's got things that just rub them the wrong way, right?
```

Comment:
Maybe a bit oversimplified?

> I mean, that goes for everyone, doesn't it? We've all got things that rub us the wrong way.

It's a minor point, admittedly. But trying to use the same colloquial structure here.